src/services/analytics/events/addressBook.ts

```diff
@@ -0,0 +1,46 @@
+import { GTM_EVENT } from '@/services/analytics/analytics'
+
+const ADDRESS_BOOK_CATEGORY = 'address-book'
+
+export const ADDRESS_BOOK_EVENTS = {
+  EXPORT: {
+    event: GTM_EVENT.CLICK,
```

Comment:
CLICK could simply be the default event.

Reply:
How would you label what is meta information?

Reply:
`event: GTM_EVENT.META,`

Reply:
As in `mobileAppPromotion.ts`?

Reply:
Same as now, just don't write `event: GTM_EVENT.CLICK,` in each event, make it the default.

Reply:
I would argue that it's more declarative as it is, no? I'll adjust it nonetheless.

Reply:
It's more verbose yes, and potentially can save someone from accidentally adding a META-type event that would be tracked as a CLICK. But also I don't think we have that many meta events.